server/controllers/ping.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2018 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an 'AS IS' BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+const log4js = require('log4js');
+const config = require('config');
+const { Gateway } = require('fabric-network');
+
+const util = require('../helpers/util');
+const walletHelper = require('../helpers/wallet');
+const ccp = require(util.getNetworkConfigFilePath('org1')); // common connection profile
+
+const gateway = new Gateway();
+const logger = log4js.getLogger('controllers - ping');
+logger.setLevel(config.logLevel);
+
+// default user and org
+const org = 'org1';
+const user = 'user1';
+const pw = 'userpw';
```

Comment:
These values should come from somewhere that is not at risk of being included in a git commit

Reply:
@eymorale Wondering if the org, user ID and password should be obtained from the network file.   Meaning for the boiler  plate we would use the admin ID and password from the network file and it is up to the user to change how we get that later on if that model is not good enough.

Reply:
@jorgedr94 I think that's a good idea that can be easily implemented for now. Makes sense that everyone will have different requirements for how to get this information and they will likely change it anyway from this boilerplate